schemas/telemetry/frecency-update/frecency-update.4.schema.json

```diff
@@ -7,6 +7,18 @@
     },
     "payload": {
       "properties": {
+        "bookmark_and_history_num_suggestions_displayed": {
+          "minimum": 0,
+          "type": "number"
+        },
+        "bookmark_and_history_rank_selected": {
+          "minimum": -1,
+          "type": "number"
+        },
+        "enter_was_pressed": {
+          "minimum": 0,
```

Comment:
It could be a boolean, true. Originally it was implemented as `enter_was_used_to_select` in which case the equivalent of `null` was necessary (in case no suggestion was selected) - something that for consistency with other attributes was represented as -1. Missed out on converting this to a boolean after the refactoring.